bin/users

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+$stdout.sync = true
+
+$LOAD_PATH << 'lib'
+
+require 'bundler/setup'
+require 'travis/api/v3/models/user'
+
+option = ARGV[1]
+
+users = if option == '--active'
+  User.where('github_oauth_token IS NOT NULL AND suspended=false').order(login: :asc)
+elsif option == '--suspended'
+  User.where(suspended: true).order(login: :asc)
+elsif option == nil
+  User.all.order(login: :asc)
+else
+  Travis.logger.error "Option #{option} not recognised."
+end
+
+users.each do |user|
```

Comment:
If the `if` statement reaches the `else`, `users` will be `true` (_I think_), and you'll get `undefined method 'each' for true:TrueClass`. Probably have to guard this somehow. One way is to add `if users`, e.g. after `end`.

Reply:
added